### ISSUE TITLE
[request] JPG DepthMaps

### ISSUE THREAD

**Issue by sfjuocekr** [alicevision/meshroom#1121]
MeshRoom currently does not handle JPG's that contain a depthmap, like the images you get from phones with a depth sensor like the HMD Nokia 9 PureView.

On Linux the only reliable way I found to extract the depth image was to use:

exiftool -DepthImage image -b > image.depth
convert -depth 8 -size 4032x3016 image.depth depth_image.jpg

These images -should- be helpful to MeshRoom? But I can't seem to get it to work, this does work with Metashape as far as I can tell. I photographed a bunch of stones next to my house, MeshRoom can't make anything out of it but with Metashape I can get a fairly decent model + texture from just using the images and from the depth maps.

I can upload the images somewhere if anyone wants to experiment!

**Comment by simogasp** [comment]
yes please if you have a small dataset you can upload it here, maybe someone will be interested in working on it. 
But for the moment the depth map is not used, it could be a very interesting feature to add. (it also depends on the quality of the depth maps)

**Comment by sfjuocekr** [comment]
http://downloads.sjoer.org/photos.tar.bz2

Some of the depthmaps are botched, when I set it to sequential it finds the most cameras in Meshroom and with Metashape it usually fails to align just one (the one that is shot near horizontally with the car in the background).

**Comment by natowi** [comment]
Similar to https://github.com/alicevision/meshroom/issues/188#issuecomment-575887872 and https://github.com/alicevision/meshroom/issues/623 (https://github.com/alicevision/meshroom/issues/255)

**Comment by CeciliaPYY** [comment]
@sfjuocekr Hello, may I ask where can I find the tutorial about sending depth maps together with rgb images into Metashape to 3d reconstruction??? Thanks a lot!!!

**Comment by natowi** [comment]
@CeciliaPYY Sorry, there is no such tutorial.
You should ask this in the Agisoft forum https://www.agisoft.com/forum/index.php?topic=9729.0
https://www.agisoft.com/forum/index.php?topic=11370.0
but it does not look like it is possible. (And I don´t see the point here, if you have Metashape you can use it to create your depth maps)

**Comment by CeciliaPYY** [comment]
@natowi Anyway, thanks again for your reply, I'll check it out by myself in related tutorial.

**Comment by sfjuocekr** [comment]
Sorry a bit late, but yea there is no such tutorial as there is no support for it.

Metashape has no support either, they all generate depth maps after resolving scene positions and thus also "depth".

**Comment by Vineg** [comment]
Hi, sfjuocekr!

Could we use your dataset as an example in the tutorial( https://agisoft.freshdesk.com/support/solutions/articles/31000162212-smart-cameras-with-depth-sensor-data-processing )?

**Comment by sfjuocekr** [comment]
Yes
